beacon-chain/state/stateutil/execution_payload_availability_root.go

```diff
@@ -0,0 +1,25 @@
+package stateutil
+
+import (
+	"fmt"
+
+	"github.com/OffchainLabs/prysm/v7/encoding/ssz"
+)
+
+// ExecutionPayloadAvailabilityRoot computes the merkle root of an execution payload availability bitvector.
+func ExecutionPayloadAvailabilityRoot(bitvector []byte) ([32]byte, error) {
```

Comment:
Note to myself, this function needs to be optimized, copying here all the bytes and chunking doesn't make sense. For smaller bitfields it does make sense, but this will have 256 chunks